client/src/components/ExportSuccess.tsx

```diff
@@ -0,0 +1,44 @@
+import { IoMdArrowBack as Arrow } from "react-icons/io";
+import { useNavigate, Link } from "react-router-dom";
+import { twMerge } from 'tailwind-merge';
+
+import H2 from "./H2";
+import Body from "./Body";
+import Button from "./Button";
+
+const ExportSuccess = ({
+  className,
+  ...rest
+}: React.HTMLAttributes<HTMLDivElement>) => {
+  const navigate = useNavigate();
+
+  return (
+    <div
+      className={twMerge('flex flex-col items-center gap-5 mx-36', className)}
+      {...rest}
+    >
+      <Button
+        icon={Arrow}
+        variant='tertiary'
+        className='absolute left-0'
+        onClick={() => navigate(-1)}
+      >
+        Back
+      </Button>
+      <H2>Export Successful!</H2>
+      <Body>
+        Congrats! Check out your new notes page :&#41;
+      </Body>
+      <div className='flex gap-5'>
+        <Link to='/' tabIndex={-1}>
+          <Button variant='secondary'>Home</Button>
+        </Link>
+        <a href='https://www.notion.so/' target='_blank'>
```

Comment:
Set `tabIndex={-1}` on this as well - what that does it removes the extra tab stop for the link; because there's a button nested inside, when a keyboard user tabs forward, it goes to the link and then the button which is redundant  since they're functionally the same thing.

I just tested it and I'm honestly not sure why the link works if you put `tabIndex={-1}` on the `a` but not on the button since the href is actually on the `a`, but I'm not going to worry about it too much